docs/faq.rst

```diff
@@ -398,6 +398,52 @@ Two mailing lists.
+* **Durability**: All transaction results are permanently preserved.
+
+ACID compliance is a complex and often contested topic. The consistency guarantee is incredibly difficult to deliver in a globally distributed database topology that involves multiple clusters, each containing many nodes.
+
+For this reason, ACID-compliant databases are usually very expensive and difficult to scale. Since SQL databases are all ACID compliant to varying degrees, they also share these downsides. 
+Some relational database systems enable ACID guarantees to be relaxed in the interest of offsetting these downsides. Achieving resilient, distributed SQL database deployments can be extraordinarily difficult and expensive.
+In contrast to SQL’s ACID guarantees, NoSQL databases provide BASE guarantees:
+
+* **Basic Availability**: Data is available most of the time, even during a partial system failure.
+* **Soft state**: Replicas are not consistent all the time.
+* **Eventual consistency**: Data will become consistent at some point in time, with no guarantee when.
+
+This is also related to the `CAP theorem <https://groups.google.com/d/forum/scylladb-users>`_, which states that in a distributed data system, only two out of the following three guarantees can be satisfied:
+
+* **Consistency**: The same response is given to all identical requests
+* **Availability**: Requests receive responses even during a partial system failure
+* **Partition** Tolerance: Operations remain intact even when some nodes are unavailable.
+
+ScyllaDB, as well as `Apache Cassandra <https://cassandra.apache.org/>`_ and other NoSQL databases, sacrifice a degree of consistency in order to increase availability. Rather than providing strong consistency, they provide `eventual consistency <https://www.scylladb.com/glossary/eventual-consistency/>`_. This means that in some cases, a read request will fail to return the result of the latest WRITE. 
+In ScyllaDB (and Apache Cassandra), consistency is tunable, meaning that for a given query, the client can specify a `Consistency Level <https://university.scylladb.com/courses/scylla-essentials-overview/lessons/architecture/topic/consistency-level-cl/>`_, which refers to the number of replicas that are required to respond to a request for it to be considered successful.  This is also known as `Tunable Consistency <https://docs.scylladb.com/stable/glossary.html#term-Tunable-Consistency>`_. 
+
+A related feature is `Lightweight Transactions <https://docs.scylladb.com/stable/using-scylla/lwt>`_ (LWT). LWT in ScyllaDB allow the client to modify data based on its current state: that is, to perform an update that is executed only if a row does not exist or contains a certain value. 
+LWT are limited to a single conditional statement, which allows an “atomic compare and set” operation. That is, it checks if a condition is true, and if so, it conducts the transaction. If the condition is not met, the transaction does not go through.
```

Comment:
Again, this paragraph, while true, does not use the word "consistency", nor "eventual" or "strong", so I think it might not be clear to the reader how it relates to the question.